src/main/java/io/debezium/connector/cassandra/CommitLogPostProcessor.java

```diff
@@ -20,7 +20,6 @@
 public class CommitLogPostProcessor extends AbstractProcessor {
 
     private static final String NAME = "Commit Log Post-Processor";
-    private static final int SLEEP_MS = 100;
```

Comment:
So the default value of this changed (now 10_000 ms), I suppose it's intentionally?

Reply:
> So the default value of this changed (now 10_000 ms), I suppose it's intentionally?

Yes, otherwise CommitLogPostProcessor will keep submitting upload job repeatedly for the same filed while it is being uploaded.